src/index.js

```diff
@@ -32,8 +32,10 @@ class CytoscapeDomNode {
             this._add_node(ev.target);
         });
 
-        for (let n of cy.nodes())
-            this._add_node(n);
+        if (cy.nodes().length){
```

Comment:
I don't get it, if `cy.nodes().length` is false then the for loop will do nothing anyway, I don't see how the extra `if` clause functionally changes the code in any way?

Reply:
For some reason when cy.nodes() returns an empty Collection, you get : 

![image](https://user-images.githubusercontent.com/17728782/144659861-7a677e8e-4888-4440-b7ac-2ab5a3045ffb.png)

Here's our "dependencies": {
    "core-js": "^3.6.5",
    "cytoscape-cola": "^2.5.0",
    "cytoscape-cxtmenu": "^3.4.0",
    "cytoscape-dom-node": "github:gpharand/cytoscape-dom-node",
    "cytoscape-edgehandles": "^4.0.1",
    "vue": "^2.6.11",
    "vue-cytoscape": "^1.0.8"
  },

Reply:
How interesting. Still doesn't make sense to me though. I'm sure you must also be dependent on some version of cytoscape? What is `cy` and `cy.nodes` and `cy.nodes.length` under these circumstances? See here https://codepen.io/mwri/pen/eYGpZgG this appears to not have a problem. Do you have a demo you can share?

Reply:
Hi, we will get back to you this week :)

Reply:
Sure, great, thanks. Your PR is obviously harmless, but it seems sufficiently impossible that it could / should make a difference I'm concerned that this is just masking something else that's going on, which could mean a bug of a slightly different nature in cytoscape-dom-node, or in your code, or Cytoscape.js, even another Cytoscape plugin you're using. We should try and understand it a little more I think.